cockatrice/src/carddatabasemodel.cpp

```diff
@@ -117,40 +117,78 @@ CardDatabaseDisplayModel::CardDatabaseDisplayModel(QObject *parent)
     setSortCaseSensitivity(Qt::CaseInsensitive);
 }
 
+
+bool CardDatabaseDisplayModel::lessThan(const QModelIndex &left, const QModelIndex &right) const {
+
+    QString leftString = sourceModel()->data(left).toString();
+    QString rightString = sourceModel()->data(right).toString();
+
+    if (leftString.compare(cardName, Qt::CaseInsensitive) == 0) {// exact match should be at top
```

Comment:
Never mind, that's fine as is

On Tue, Mar 3, 2015, 10:55 AM poixen <notifications@github.com> wrote:

> In cockatrice/src/carddatabasemodel.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/796#discussion_r25698311>:
>
> > @@ -117,40 +117,78 @@ CardDatabaseDisplayModel::CardDatabaseDisplayModel(QObject *parent)
> >      setSortCaseSensitivity(Qt::CaseInsensitive);
> >  }
> >
> > +
> > +bool CardDatabaseDisplayModel::lessThan(const QModelIndex &left, const QModelIndex &right) const {
> > +
> > +    QString leftString = sourceModel()->data(left).toString();
> > +    QString rightString = sourceModel()->data(right).toString();
> > +
> > +    if (leftString.compare(cardName, Qt::CaseInsensitive) == 0) {// exact match should be at top
>
> 'æ' is 'ae' with card names. Im not sure its ever been used.
> Not sure what you want me to do with that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/796/files#r25698311>.
>